<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>893</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/img/icon.png</normaloff>:/img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 245, 203);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255, 255);
border-radius : 30px;</string>
     </property>
     <property name="text">
      <string>Aisleschool レッスン サーバー アカウント マネージャー</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: &quot;&quot;;
background-color: rgba(255, 255, 255,180);
font-size : 13pt;
border-radius : 7px;

</string>
     </property>
     <property name="title">
      <string>【VPSログイン】</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255,0);</string>
        </property>
        <property name="text">
         <string>ユーザー名</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>パスワード</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="log_domain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="log_txt3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
max-height : 35px;
min-height : 35px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="log_name">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>ドメイン</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="cp3_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>コピー3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="log_txt1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
max-height : 35px;
min-height : 35px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="log_pass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="cp1_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>コピー1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="cp2_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>コピー2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="log_pem">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="log_url_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>ファイルを開く</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="log_txt2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
max-height : 35px;
min-height : 35px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>pemファイルのURL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="log_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="help_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightgrey;
background-color: grey;
color : white;
</string>
        </property>
        <property name="text">
         <string>(?)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="add_group">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,180);
font-size : 13pt;
border-radius : 7px;
background-image: none</string>
     </property>
     <property name="title">
      <string>【ユーザーを追加】</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="add_domain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="add_txt">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
max-height : 35px;
min-height : 35px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>パスワード</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>ドメイン</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="add_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="cp1_btn_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>コピー</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="add_pass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="add_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>ユーザー名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="del_group">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255,180);
font-size : 13pt;
border-radius : 7px;
</string>
     </property>
     <property name="title">
      <string>【ユーザーの削除】</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>ユーザー名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>ドメイン</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="del_txt">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
max-height : 35px;
min-height : 35px</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="cp1_btn_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>コピー</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>パスワード</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="del_domain">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="del_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>生成</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="del_name">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="del_pass">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="list_btn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius : 5px;
border : 3px solid lightblue;
background-color: qlineargradient(spread:pad, x1:0.483, y1:1, x2:0.5, y2:0, stop:0 rgba(57, 111, 189, 255), stop:0.295455 rgba(201, 212, 236, 255), stop:0.505682 rgba(216, 223, 236, 255), stop:0.710227 rgba(203, 222, 255, 255), stop:1 rgba(52, 101, 255, 255));
</string>
        </property>
        <property name="text">
         <string>ユーザーリストコマンドのコピー</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>log_name</tabstop>
  <tabstop>log_domain</tabstop>
  <tabstop>log_url_btn</tabstop>
  <tabstop>log_pass</tabstop>
  <tabstop>log_btn</tabstop>
  <tabstop>cp1_btn</tabstop>
  <tabstop>cp2_btn</tabstop>
  <tabstop>cp3_btn</tabstop>
  <tabstop>add_name</tabstop>
  <tabstop>add_domain</tabstop>
  <tabstop>add_pass</tabstop>
  <tabstop>add_btn</tabstop>
  <tabstop>cp1_btn_2</tabstop>
  <tabstop>del_name</tabstop>
  <tabstop>del_domain</tabstop>
  <tabstop>del_pass</tabstop>
  <tabstop>del_btn</tabstop>
  <tabstop>cp1_btn_3</tabstop>
  <tabstop>log_pem</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
